fairmq/zeromq/Common.h

```diff
@@ -87,6 +92,16 @@ inline int HandleErrors(const std::string& id)
     }
 }
 
+__FAIRMQ_ALWAYS_INLINE auto ZSend(void* socket, UMsg&& msg, int flags)
+{
+    return zmq_send(socket, msg.Data(), msg.Size(), flags);
+}
+
+__FAIRMQ_ALWAYS_INLINE auto ZSend(void* socket, ZMsg&& msg, int flags)
+{
+    return zmq_msg_send(msg.Msg(), socket, flags);
+}
```

Comment:
I can also remove those. They were more complicated earlier, but got simplified to a one-liner, which makes the `ZSend` not that useful any more.

Reply:
Great, I wanted to ask about these too!